internal/pkg/process/discover.go

```diff
@@ -84,7 +84,7 @@ func (a *Analyzer) findProcessID(target *TargetArgs) (int, error) {
 
 	for {
 		dirs, err := proc.Readdir(15)
-		if err == io.EOF {
+		if errors.Is(err, io.EOF) {
```

Comment:
The current code will always work as otherwise it would be a breaking change of https://pkg.go.dev/os#File.Readdir.

I think we can keep as it is.

Reply:
Returning a wrapped `io.EOF` wouldn't be a breaking change in the Go ecosystem. What's the downside of making this change?